### PR TITLE
Py doc utils client fix

### PR DESCRIPTION
**Pull request by theolebever**
Fix changes asked on this PR #117. 

Makes changes on : 
- Fix comment format not recognised by PyDoc
- Removal of examples in order to add them all at once later
- More details on some classes

### CODE REVIEW

**Review by JoFrost — APPROVED** [review]
To be frank, I would have preferred to have more examples instead of deleting them completely. Other than that, it looks good.

**Comment by JoFrost** [comment]
@theolebever Can you please fix the formatting issue?

**Comment by theolebever** [comment]
Done @JoFrost

**Review by dhalf — CHANGES_REQUESTED** [review]
Use a single docstring style and be consistent
https://google.github.io/styleguide/pyguide.html#381-docstrings

File: client/src/bastionlab/polars/client.py [hunk 1]
```diff
@@ -23,15 +23,13 @@
 
 
 class BastionLabPolars:
-    """
-    Main BastionLabPolars API class.
+    """Main BastionLabPolars API class.
```
Comment (JoFrost) [inline client/src/bastionlab/polars/client.py:26]: As the Torch part is now part of BastionLab, this phrase is wrong. I would rather specify that it's the polars related functions, or something like that.

**Review by JoFrost — CHANGES_REQUESTED** [review]
Looks fine, but please change the Polars class description.

File: client/src/bastionlab/polars/client.py (hunk 1 above)
Comment (theolebever) [reply client/src/bastionlab/polars/client.py:26]: I've now updated the description of the BastionLabPolars class according to the integration of torch related functionnalities.

Comment (JoFrost) [reply client/src/bastionlab/polars/client.py:26]: Thank you, that's way better. However, the torch client lacks details. Please do the same for the torch client.

**Review by dhalf — CHANGES_REQUESTED** [review]
This PR seems to reintroduce classes and functions that were removed on master
Please sync with master to be sure everything's all right

**Review by JoFrost — APPROVED** [review]

**Pull request merged**